src/app/(contents)/post/[detailId]/PostDetail.tsx

```
@@ -0,0 +1,203 @@
+    const diff = dayjs().diff(dayjs(postDetail.createdAt), "hour")
+
+    useEffect(() => {
+        getPostDetail(Number(id))
+    }, [id, doEdit])
+
+    // HTML 파싱 및 img 태그 변환 함수
+    const parseHtmlWithNextImage = (htmlContent: string) => {
+        const options: HTMLReactParserOptions = {
+            replace: (domNode) => {
+                if (domNode instanceof Element && domNode.name === "img") {
+                    const { src, alt = "", width, height, style = "" } = domNode.attribs
```

ESLint

[error] 'width' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)